bindings/ocaml/SharedFunctors.ml

```diff
@@ -113,7 +113,7 @@ module Make_EdDSA_generic (C: Buffer)
     (* Hacl.Ed25519.sign_expanded *)
     assert (C.size ks = 96);
     assert (C.size signature = 64);
-    assert (C.size msg + 64 <= max_size_t);
+    assert Z.(of_int (C.size msg) + ~$64 <= max_size_t);
```

Comment:
I suspect these overflow on 32-bit ARM. Have you gone through a general cleanup pass throughout all the bindings?

Reply:
Yes, that was the reason I changed it throughout the bindings.